### DISCUSSION TITLE
TOML to YAML conveter?

### DISCUSSION THREAD

**Discussion by smit-io** [CorentinTh/it-tools#465] [Ideas]
Hi,

Just like YAML to JSON converter, can we please also have a TOML to YAML converter. Thanks.

**Comment by smit-io** [comment]
Something like this: https://transform.tools/toml-to-yaml

**Comment by CorentinTh** [comment]
Hi @smit-io 
Thank you for the suggestion 🙏🏻 
It's done, 4 new format converters:

* **YAML to TOML** : https://it-tools.tech/yaml-to-toml
* **JSON to TOML** : https://it-tools.tech/json-to-toml
* **TOML to JSON** : https://it-tools.tech/toml-to-json
* **TOML to YAML** : https://it-tools.tech/toml-to-yaml
